IzvorniKod/src/main/java/com/educhat/backend/DTO/PostCreateDTO.java

```diff
@@ -16,6 +16,5 @@ public class PostCreateDTO {
     private String postContent;
     private Long subjectId;
     private List<String> images;
-    private String link;
```

Comment:
Što ipak nema linka u postu?

Reply:
@Hrvoje8 trebao bi nam link parametar za dodavanje nekih vanjskih poveznica, mozemo ga nazvad externalLink da je mozda razumljivije